### PR TITLE
OTWO-6809 Add script to create Sbom jobs & UI to view Sbom data

### PR DESCRIPTION
**Pull request by Priya5**

### CODE REVIEW

**Comment by Priya5** [comment]
retest this please

File: db/migrate/20220913135438_create_project_sbom.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateProjectSbom < ActiveRecord::Migration[5.2]
+  def change
+    create_table :project_sboms do |t|
+      t.integer :project_id
+      t.integer :code_location_id
+      t.jsonb :sbom_data
+    end
+    add_index :project_sboms, :sbom_data, using: :gin
```
Comment (notalex) [inline db/migrate/20220913135438_create_project_sbom.rb:10]: We will not be searching this table by sbom_data, so no index is required.

**Review by notalex — APPROVED** [review]

**Comment by notalex** [comment]
retest this please

**Comment by notalex** [comment]
retest this please

**Review by notalex — APPROVED** [review]

**Pull request merged**